lib/private/help-exec-machine-instance.js

```diff
@@ -806,7 +806,7 @@ module.exports = function helpExecMachineInstance (liveMachine) {
       } catch (e) {
 
         // Split the error we got into lines, and remove the first one (the actual error message)
-        e.stack = e.stack.split('\n');
+        e.stack = stripStackMessage(e).split('\n');
         var msg = e.stack.shift();
```

Comment:
But 👍 about the `e.name`, I didn't realize until yesterday afternoon that that wasn't part of `e.message`.